docs/topics/mta-rn-known-issues-7-0-2.adoc

```diff
@@ -11,12 +11,14 @@
 
 .Failure to connect to Jira server using basic authentication
 
-In {ProductShortName} 7.0.2, connecting to a Jira server using basic authentication, meaning _username_ and _password_, fails. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2427[(MTA-2427)]  
+In {ProductShortName} 7.0.2, connecting to a Jira server using basic authentication, meaning _username_ and _password_, fails. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2427[(MTA-2427)]
```

Comment:
I think we need to change the version to 7.1 now .

This issue is scheduled to be resolved in {ProductShortName} 7.1